pkg/injector/patcher/sidecar.go

```diff
@@ -22,6 +22,7 @@ import (
 	corev1 "k8s.io/api/core/v1"
 
 	injectorConsts "github.com/dapr/dapr/pkg/injector/consts"
+	"github.com/dapr/dapr/pkg/security"
```

Comment:
pkg/injector just went through a refactoring to make it have minimal dependencies on the rest of Dapr. THis is because it can be used as a library outside of Dapr. Looks like pulling in pkg/security brings in a lot, including transitive dependencies such as pkg/diagnostics. What is this package used for in the injector, and can an alternative be found please?